### PR TITLE
Update deployment-install to run cross-platform (incl Windows)

### PR DESCRIPTION
**Pull request by dan-cheek-perficient**
Added new npm dependencies 'tar' and 'mkdirp'

### CODE REVIEW

File: src/common/deployment-install.js [hunk 1]
```diff
@@ -39,10 +41,16 @@ for(let action in serverless.functions) {
 // Provision dependencies
 for(let dependency of dependencies) {
     // Download dependencies, unpack them and install their production dependencies
-    child_process.execSync(`TARBALL=$(npm pack '${dependency}')` +
-        ` && mkdir -p '${dependency}'` +
-        ` && tar xvf $TARBALL -C '${dependency}' --strip 1` +
-        ` && rm $TARBALL` +
-        ` && cd '${dependency}'` +
-        ` && npm install --only=production`, {stdio: 'inherit'});
+    child_process.execSync(`npm pack ` + dependency + ` > tarname.tmp`);
```
Comment (herzog31) [inline src/common/deployment-install.js]: This part can be simplified to:

```
let tarball = child_process.execSync(`npm pack ${dependency}`).toString().trim();
mkdirp.sync(dependency);
tar.x({
    file: tarball,
    C: dependency,
    strip: 1,
    sync: true
});
fs.unlinkSync(tarball);
child_process.execSync(`cd ${dependency} && npm install --only=production`);
```

File: src/common/package.json [hunk 2]
```diff
@@ -11,8 +11,10 @@
   "dependencies": {
     "@adobe/commerce-cif-common": "0.1.17",
     "@adobe/commerce-cif-model": "0.1.123",
+    "mkdirp": "^0.5.1",
```
Comment (cjelger) [inline src/common/package.json]: Please remove the `^` in the version of packages: we use fixed versions to make sure all the builds are always the same for everybody.

File: package.json [hunk 3]
```diff
@@ -40,17 +40,17 @@
     "all": true
   },
   "scripts": {
-    "pack": "$(npm bin)/lerna exec -- npm pack",
```
Comment (cjelger) [inline package.json]: Can you please also fix the other `package.json` files in the repo? Should be a few in `src` and one in `test`.

**Comment by herzog31** [comment]
Will be fixed in https://github.com/adobe/commerce-cif-magento/pull/40.

**Pull request closed without merging**